src/unity/lib/annotation/image_classification.cpp

```diff
@@ -95,11 +95,17 @@ annotate_spec::Annotations ImageClassification::getAnnotations(size_t start,
 
     label->imageclassificationlabel();
```

Comment:
Aside: isn't this a no-op?

Reply:
Not if it's in a `oneof` that hasn't yet had any of its members selected. `oneofs` are really zero-of-until-accessed... accessing one member makes it the one. By design in proto3.

Reply:
Strange... the implementation appears to be a const method that I don't believe sets the one-of state: https://github.com/apple/turicreate/blob/5e960c084f236195a55b60bb7ba3ab0e650c87d5/src/unity/lib/annotation/build/format/cpp/annotate.pb.cc#L1984

If the intent is to set the one of, could we use the more idiomatic `mutable_imageclassificationlabel` instead? We can see that that method sets the field explicitly:
https://github.com/apple/turicreate/blob/5e960c084f236195a55b60bb7ba3ab0e650c87d5/src/unity/lib/annotation/build/format/cpp/annotate.pb.cc#L1993

Reply:
Actually, that's probably what was intended. I'll change it.